pls_cli/please.py

```diff
@@ -131,6 +132,45 @@ def quotes(show: bool = True) -> None:
     )
 
 
+class language(str, Enum):
+    english = "english"
```

Comment:
I think it's better to use the short form, like `en` for english and `es` for spanish, what do you think?